### ISSUE TITLE
Hyperparameters in aerial scene

### ISSUE THREAD

**Issue by LaFeuilleMorte** [zju3dv/PGSR#54]
Hi, I noticed that your multi view nearest cam selection strategy was : select those cameras with angle between (0, 30) and distance between (0.01, 1.5) from the candidate cameras. Are these settings suitable for the large outdoor scenes? For example, My dataset is an aerial scene whose images are from a drone with 5 cameras shot from 5 different angles. And I also observed very small number of nearest cameras in my "mulit_view.json". Which means the multi-view loss can hardly work in my scene. Hope for your reply!

**Comment by LaFeuilleMorte** [comment]
> Hi, the multi-view nearest camera selection strategy may require choosing appropriate thresholds based on the actual scenario, as the default parameters may not always be suitable. You can check whether the strategy is appropriate by examining the final file "multi_view.json."

I still had some questions that may needs your instruction, In the nearest cam selection stage:

![Screenshot 2024-10-28 at 14 40 28](https://github.com/user-attachments/assets/3d67fd99-a747-40e4-b26a-7e0dd03a050d)

Here the nearest camera will take angle offset and distance offset into consideration:

But the camera distance was an absolute distance between two camera centers. Due to the scale of different scenes. For example, when in indoor scenes, the camera might be much closer than in outdoor scenes. So I wonder if it would be better to use relative camera offsets. For example, divided the offset by the bounding box of the whole scene. Looking forward to your reply!

**Comment by danpeng2** [comment]
Yes, if the entire scene size can be considered as the basis for adjustment, the parameters will be more universal.
